esmf-semantic-aspect-meta-model/src/test/resources/samm_2_0_0/time-series/org.eclipse.esmf.test/1.0.0/TimeSeriesTestEntity.ttl

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2021 Robert Bosch Manufacturing Solutions GmbH
+#
+# See the AUTHORS file(s) distributed with this work for additional
+# information regarding authorship.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+@prefix : <urn:org.eclipse.esmf.samm.test:1.0.0#> .
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:TimeSeriesTestEntity a samm:Aspect;
+   samm:name "TimeSeriesTestEntity";
+   samm:properties (:timeseries);
+   samm:operations ();
+   samm:events ().
+:timeseries a samm:Property;
```

Comment:
Please format the file a little bit nicer, with an empty line between element definitions (see other test files).